<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view_proc_order</class>
 <widget class="QWidget" name="view_proc_order">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="orderList"/>
      </item>
      <item>
       <widget class="QTextBrowser" name="orderDetailBrowser"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="minimumSize">
         <size>
          <width>316</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>tips：订单状态更新顺序：未付款-&gt;已付款-&gt;已完成</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateStatusButton">
        <property name="text">
         <string>更新订单状态</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refuseButton">
        <property name="text">
         <string>拒绝此订单</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
